src/App.jsx

```diff
@@ -0,0 +1,98 @@
+import React, {useState} from "react";
+
+import FileField from "./components/FileField";
+import CheckBox from "./components/CheckBox";
+
+import PreviewProcessedData from "./components/PreviewProcessedData";
+
+import {createExcelFile, createExcelFile2, downloadFinalXLSX} from "./utils.js"
+
+import { BsDownload } from "react-icons/bs";
+
+const App = () => {
+  const [fileData, setFileData] = useState(null);
+  const [processedData, setProcessedData] = useState(null);
+  const [checkbox, setCheckbox] = useState(null);
+
+  let processedDataFromMultipleFiles = [];
+  
+  const getProcessedData = (initialData) => {
+  
+    initialData.forEach(el => processedDataFromMultipleFiles.push(JSON.parse(el)
+      .data
+      .map(({r1, r2, ils, fzl, izl, ozl}) => ({
+            ils: {
+              value: ils || "",
+              name: "Страховой номер"
+            }, 
+            fzl: {
+              value: fzl || "",
+              name: "Фамилия"
+            },
+            izl: {
+              value: izl || "",
+              name: "Имя"
+            },
+            ozl: {
+              value: ozl || "",
+              name: "Отчество"
+            },
+            dto1: {
+              value: r1[0].dto1 || "",
+              name: "Дата увольнения"
+            },
+            dfr21: {
+              value: r2[0].dfr21 || "",
+              name: "Дата приема - перевода"
+            },
+          }))
+        ))
+
+    return processedDataFromMultipleFiles;
+  }
+
+  const createExcelFileDependingOnCheckboxState = (data) => {
```

Comment:
Make it like so:

`() => downloadFinalXLSX(createExcelFile(data), checkbox)`

remove doubling of `createExcelFile2`

Reply:
done